### PR TITLE
Ported the BaselineCorrectionFunction class from neurolab-desktop

### PR DESCRIPTION
**Pull request by RohitJain1103**
Fixes #17 partially.

**Changes**: 
Ported the independent BaselineCorrectionFunction class and placed it in the appropriate subpackage(io.neurolab.tools) following the neurolab-desktop's standards.

**Checklist**:
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications are done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: [Compress the `app-debug.apk` file into a `<feature>.rar` or `<feature>.zip` file and upload it here]

### CODE REVIEW

**Comment by RohitJain1103** [comment]
@CloudyPadmal @mariobehling @iamareebjamal could you please review the PR?

File: build.gradle [hunk 1]
```diff
@@ -28,7 +28,7 @@ task clean(type: Delete) {
 
 ext {
 
-    minSdkVersion = 16
+    minSdkVersion = 24
```
Comment (jddeep) [inline build.gradle]: We should support from API level 16

**Review by suresh-gandhi — COMMENTED** [review]
@RohitJain1103 I would suggest you to annotate your methods/classes(which enforces an upgrade) with the TargetApi.

**Comment by RohitJain1103** [comment]
> @RohitJain1103 I would suggest you to annotate your methods/classes(which enforces an upgrade) with the TargetApi.

 @suresh-gandhi Okay. Thanks.

**Review by suresh-gandhi — APPROVED** [review]

File: app/src/main/java/io/neurolab/tools/BaselineCorrectionFunction.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package io.neurolab.tools;
+
+import android.annotation.TargetApi;
+
+import java.util.function.Function;
+
+import io.neurolab.MathBasics;
+
+@TargetApi(24)
```
Comment (jddeep) [inline app/src/main/java/io/neurolab/tools/BaselineCorrectionFunction.java:9]: This makes this complete class unfunctional and not usable below API 24. Please change.

**Review by jddeep — CHANGES_REQUESTED** [review]

**Comment by Dev-Aslan** [comment]
Hi I am Devbrat From India
I Also want to contribute to this project, can anyone please tell me where to start from?

**Comment by RohitJain1103** [comment]
@Dev-Aslan Please join the following gitter chat - https://gitter.im/fossasia/fossasia . You can get started with any project. Just start with a small issue and try to fix it.

**Pull request merged**